freqtrade/strategy/interface.py

```diff
@@ -278,7 +278,8 @@ def stop_loss_reached(self, current_rate: float, trade: Trade, current_time: dat
         # evaluate if the stoploss was hit
         if self.stoploss is not None and trade.stop_loss >= current_rate:
             selltype = SellType.STOP_LOSS
-            if trailing_stop:
+            # Trailing stop (and rate did move)
+            if trailing_stop and trade.open_rate != trade.max_rate:
```

Comment:
It would be nice if you add a comment explaining what `trade.open_rate != trade.max_rate`is necessary

Reply:
line 281? "rate did move" ?